cnd/cnd.toolchain.ui/src/org/netbeans/modules/cnd/toolchain/ui/options/ToolsPanelController.java

```diff
@@ -26,12 +26,14 @@
 import org.openide.util.HelpCtx;
 import org.openide.util.Lookup;
 
-@OptionsPanelController.SubRegistration(
-    id=CndUIConstants.TOOLS_OPTIONS_CND_TOOLS_ID,
-    location=CndUIConstants.TOOLS_OPTIONS_CND_CATEGORY_ID,
-    displayName="#TAB_ToolsTab", // NOI18N
-    position=100
-)
+// Overriden with a manually generated layer.xml file.
+// See https://issues.apache.org/jira/browse/NETBEANS-6372
+//@OptionsPanelController.SubRegistration(
+//    id=CndUIConstants.TOOLS_OPTIONS_CND_TOOLS_ID,
+//    location=CndUIConstants.TOOLS_OPTIONS_CND_CATEGORY_ID,
+//    displayName="#TAB_ToolsTab", // NOI18N
+//    position=100
+//)
```

Comment:
See comment in `CodeAssistancePanelController.java`

Reply:
On its way.